boost-maven/README.md

```diff
@@ -0,0 +1,58 @@
+# Boost Maven Plugin Prototype
+
+### What is Boost?
+
+This is a prototype Maven plugin to package a fully configured Java EE or MicroProfile application with a target runtime.
+
+When added to your pom.xml, the plugin will
+
+1. Install the desired target runtime.
+2. Create a server.
+3. Install the application to the server.
+4. Configure the server appropriatly for the application.
+5. Package the server and application into a runnable jar.
+
+### Build the Boost Maven Plugin
+
+1. `git clone git@github.com:OpenLiberty/boost.git`
+2. `boost-maven.sh`
+3. `mvn clean install`  (To run integration tests, add the -Pit parameter)
+
+### Use the Boost Maven plugin in your Application Maven project 
+
+#### Try it!
+
+Kick the tires of Boost with zero configuration:
+
+* Produce a runnable uber jar for your app:
+    * `mvn clean package boost:boost-maven-plugin:0.1:package`
+    * `java -jar target/<application name>.jar`
+
+#### Quick start - uber jar
+
+1. Add the following to your project pom.xml
+    ```xml
+      <plugin>
+            <groupId>boost</groupId>
+            <artifactId>boost-maven-plugin</artifactId>
+            <version>0.1</version>
```

Comment:
Change to version 0.1.3-SNAPSHOT

Reply:
scott gave me some direction to remove the uber jar info, so took this out

Reply:
I don't understand. The pom.xml still needs to include the boost-maven-plugin.